cool-core/src/test/java/com/nus/cool/core/refactor/ProcessorTest.java

```diff
@@ -0,0 +1,107 @@
+package com.nus.cool.core.refactor;
+
+import java.io.File;
+import java.io.IOException;
+import java.nio.file.Paths;
+import java.util.ArrayList;
+import java.util.HashMap;
+
+import org.testng.Assert;
+import org.testng.annotations.BeforeTest;
+import org.testng.annotations.DataProvider;
+import org.testng.annotations.Test;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.nus.cool.core.cohort.refactor.CohortProcessor;
+import com.nus.cool.core.cohort.refactor.storage.CohortRet;
+import com.nus.cool.core.io.readstore.CubeRS;
+import com.nus.cool.core.util.config.CsvDataLoaderConfig;
+import com.nus.cool.core.util.config.DataLoaderConfig;
+import com.nus.cool.model.CoolLoader;
+import com.nus.cool.model.CoolModel;
+
+public class ProcessorTest {
+    private final String cubeRepo = "../datasetSource";
+    private final String rootPath = "../datasets";
+    private final String tableName = "table.csv";
+    private final String configName = "table.yaml";
+    private CoolModel coolModel;
+
+    private final String queryName = "query.json";
+    private final String resultName = "query_result.json";
+
+    @BeforeTest
+    public void startUp() throws IOException{
```

Comment:
Neet to format this unit test file. i.e., add `Setup` and `TearDown` functions and logger.

Reply:
Update